strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/security/AuthoritiesProvider.java

```diff
@@ -39,88 +32,32 @@
-        {
-            authorities.addAll(fullAuthorities);
-        }
-
-        // add all privileges from etc/conf/strongbox-authorization.yaml for any role that defines there
-        configuredRoles.forEach(role ->
-                                {
-                                    if (role.getName()
-                                            .equalsIgnoreCase(roleName))
-                                    {
-                                        role.getPrivileges()
-                                            .forEach(
-                                                    privilegeName -> authorities.add(
-                                                            new SimpleGrantedAuthority(privilegeName.toUpperCase())));
-                                    }
-                                });
-
-        try
-        {
-            Roles configuredRole = Roles.valueOf(roleName);
-            authorities.addAll(configuredRole.getPrivileges());
-        }
-        catch (IllegalArgumentException e)
-        {
-            logger.warn("Unable to find predefined role by name " + roleName);
-        }
-
-        return authorities;
-    }
-
-    private void populate(Set<GrantedAuthority> fullAuthorities,
-                          Set<Role> configuredRoles)
-    {
-        AuthorizationConfig authorizationConfig = authorizationConfigService.get();
-
-        authorizationConfig.getRoles()
-                           .forEach(
-                                   role -> role.getPrivileges()
-                                               .forEach(
-                                                       privilegeName -> fullAuthorities.add(
-                                                               new SimpleGrantedAuthority(privilegeName.toUpperCase())
-                                                       )
-                                               )
-                           );
-
-        configuredRoles.addAll(authorizationConfig.getRoles());
-    }
-
     public Set<Role> getAssignableRoles()
     {
-        ImmutableSet.Builder builder = ImmutableSet.<Role>builder()
-                                               .addAll(authorizationConfigService.get()
-                                                                                 .getRoles())
-                                               .addAll(Arrays.stream(Roles.values())
-                                                             .map(r -> new RoleDto(r.name(), r.getDescription()))
-                                                             .map(Role::new)
-                                                             .collect(Collectors.toSet()));
-        return builder.build();
+        return authorizationConfigService.get().getRoles();
+    }
+    
+    public RoleData getRuntimeRole(String name) {
+        Role role = authorizationConfigService.get()
+                                         .getRoles()
```

Comment:
Cosmetic: Could you please re-align the chained calls? Thanks! :)

Reply:
fixed